src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2
+spring.jpa.defer-datasource-initialization=true
 
+
```

Comment:
Think we need to delete this since the main branch uses a different configuration